test/functional/http/get/node.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+var node = require('../../../node.js');
+var http = require('../../../common/httpCommunication.js');
+
+describe('GET /api/node/constants', function () {
+
+	var constantsResponse;
+
+	before(function (done) {
+		http.get('/api/node/constants', function (err, res) {
+			constantsResponse = res.body;
+			done();
+		});
+	});
+
+	it('should return a result containing build = ""', function () {
```

Comment:
I think the descriptions here are redundant. We can have just one it block with all the expectations.